src/index.js

```diff
@@ -313,6 +296,43 @@ follow these steps, YOUR APP MAY BREAK.`);
     Firestore.log('Firestore', null, 'Initialized Firestore');
   }
 
+  /**
+   * Specifies custom settings to be used to configure the `Firestore`
+   * instance. Can only be invoked once and before any other Firestore method.
+   *
+   * If settings are provided via both `settings()` and the Firestore
```

Comment:
I think you can code format `Firestore` here.

Reply:
Done (here and in the `.d.ts` file).